app/src/main/java/org/simple/clinic/util/TimeFunctions.kt

```diff
@@ -23,3 +24,10 @@ fun Instant.toLocalDateAtZone(zone: ZoneId): LocalDate =
     atZone(UTC)
         .withZoneSameInstant(zone)
         .toLocalDate()
+
+fun LocalDate.toUtcInstant(clock: Clock): Instant {
```

Comment:
This should take in the `UserClock` instead of a generic `Clock`, right? This is basically meant to convert the local time in the user's timezone to the equivalent UTC `Instant`.

Reply:
The test can change to create two different `TestUserClock` instances with different zone offsets.